android/app/src/main/kotlin/org/whegba/wh_covid19/WHApplication.kt

```diff
@@ -0,0 +1,13 @@
+package org.whegba.wh_covid19
```

Comment:
Kotlin code? Again I assume this is to implement the Android platform side of it?

Reply:
yep see my comment above about native Android sentry init